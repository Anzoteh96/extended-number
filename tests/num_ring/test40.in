239214828766941676164140099947761719682779776297997336349704541616874756578016791342932755539828683110398485201062440218
-496337255891407966347183247447727534663628510374827365329488577152424785699203380464499816528648563471647149548869602449
-722724871891342567797663170955684172671977167120788290653597665817317955214485846622898967421371558320658339608339011201
