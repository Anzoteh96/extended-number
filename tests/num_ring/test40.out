Enter the number base b:
Enter two numbers n and m:
b is 239214828766941676164140099947761719682779776297997336349704541616874756578016791342932755539828683110398485201062440218
n mod b is 221307230409417062145237052395557624384710818519164643719625047698199484034846993564298450090837485859548306054317718205
m mod b is 234134443176424136858897228835362706059141938071201054745220500650181071097581318748832054737943174120935601195910749671
n + m is 216226844818899522839994181283158610761072980292368362115141006731505798554411520970197749288951976870085422049166027658
n - m is 226387615999934601450479923507956638008348656745960925324109088664893169515282466158399150892722994849011190059469408752
n * m is 86227173639535539478565010487936233016257505716512687941815909730282624111439331258475848206631711367722438280665659531
n ** m is 24889052940128009230596671545014570176712632931070283698398714793297209823418887666514137187861643531632082326175762503
